sourcegraph/sourcegraph
# Remove old Cody Web completely

closes: https://linear.app/sourcegraph/issue/SRCH-622/remove-old-cody-web-entirely

Removes the old Cody Web completely.

## Test plan

- sg start
- visit /cody/chat and you should see the new unified panel Cody Web without having to turn on any feature flags.
- check the blob view sidebar for the same.
## Changelog

- The new Cody Web is enabled by default and the older version of Cody Web is completely removed from both standalone chat and sidebar.
